<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Crapinfo</class>
 <widget class="QWidget" name="Crapinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">LogDoctor - Infos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/logo/logo/logdoctor.svg</normaloff>:/logo/logo/logdoctor.svg</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox_Infos">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabSpacing">
      <number>4</number>
     </property>
     <widget class="QWidget" name="page_LogDoc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>835</width>
        <height>521</height>
       </rect>
      </property>
      <attribute name="label">
       <string notr="true">LogDoctor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_LogDoc">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaContents_LogDoc">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>835</width>
            <height>521</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="0" colspan="3">
            <spacer name="spacer_LogDoc_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>788</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QFrame" name="frame_LogDoc_1">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>303</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="button_LogDoc_Logo">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>280</width>
                  <height>280</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>280</width>
                  <height>280</height>
                 </size>
                </property>
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="icon">
                 <iconset resource="../../resources/resources.qrc">
                  <normaloff>:/logo/logo/logdoctor.svg</normaloff>:/logo/logo/logdoctor.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>256</width>
                  <height>256</height>
                 </size>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_LogDoc_Info">
                <property name="minimumSize">
                 <size>
                  <width>400</width>
                  <height>0</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_Version">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>32</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>32</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>16</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Version</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_Verion_Number">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>32</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>32</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>13</pointsize>
                    </font>
                   </property>
                   <property name="toolTip">
                    <string>Currently installed version of the software</string>
                   </property>
                   <property name="text">
                    <string notr="true"/>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignHCenter|Qt::AlignTop</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="Line" name="gline_Version">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_Links">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>32</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>32</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>15</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Repository links</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inLine_Links_Github">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <italic>true</italic>
                    </font>
                   </property>
                   <property name="acceptDrops">
                    <bool>false</bool>
                   </property>
                   <property name="toolTip">
                    <string notr="true">GitHub</string>
                   </property>
                   <property name="text">
                    <string notr="true">https://github.com/elB4RTO/LogDoctor</string>
                   </property>
                   <property name="frame">
                    <bool>false</bool>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inLine_Links_Disroot">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <italic>true</italic>
                    </font>
                   </property>
                   <property name="acceptDrops">
                    <bool>false</bool>
                   </property>
                   <property name="toolTip">
                    <string notr="true">Disroot</string>
                   </property>
                   <property name="text">
                    <string notr="true">https://git.disroot.org/elB4RTO/LogDoctor</string>
                   </property>
                   <property name="frame">
                    <bool>false</bool>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inLine_Links_Gitlab">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <italic>true</italic>
                    </font>
                   </property>
                   <property name="acceptDrops">
                    <bool>false</bool>
                   </property>
                   <property name="toolTip">
                    <string notr="true">GitLab</string>
                   </property>
                   <property name="text">
                    <string notr="true">https://gitlab.com/elB4RTO/LogDoctor</string>
                   </property>
                   <property name="frame">
                    <bool>false</bool>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="spacer_LogDoc_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <spacer name="spacer_LogDoc_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>32</width>
               <height>408</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="3">
            <spacer name="spacer_LogDoc_1">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>788</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_Paths">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>835</width>
        <height>521</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Paths</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_Paths_Paths">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_Executable">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>The path of the executable file</string>
            </property>
            <property name="text">
             <string>Executable</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_Paths_Executable">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <widget class="QWidget" name="scrollAreaContents_Paths_Executable">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>815</width>
               <height>58</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLineEdit" name="inLine_ExecPath">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="spacer_Paths_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>32</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_ConfigFile">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>The path where the configuration file gets saved and searched in</string>
            </property>
            <property name="text">
             <string>Configuration file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_Paths_ConfigFile">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <widget class="QWidget" name="scrollAreaContents_Paths_ConfigFile">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>815</width>
               <height>58</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLineEdit" name="inLine_ConfPath">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="spacer_Paths_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>32</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_AppData">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>The path where the application searches for extra resources</string>
            </property>
            <property name="text">
             <string>Application data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_Paths_AppData">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <widget class="QWidget" name="scrollAreaContents_Paths_AppData">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>815</width>
               <height>58</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLineEdit" name="inLine_DataPath">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="spacer_Path_1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>88</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
